app/multitenant/aws_collector.go

```diff
@@ -355,6 +355,19 @@ func calculateReportKey(rowKey, colKey string) (string, error) {
 	return fmt.Sprintf("%x/%s", rowKeyHash.Sum(nil), colKey), nil
 }
 
+func withBackoff(f func() error) error {
+	retries := 0
+	backoff := 50 * time.Millisecond
+	err := f()
+	for err != nil && retries <= 5 {
+		time.Sleep(backoff)
+		err = f()
+		retries += 1
+		backoff *= 2
```

Comment:
if it's easy, maybe add some small random variance here for thundering herd protection?